http-server.ts

```diff
@@ -0,0 +1,35 @@
+import Koa = require('koa');
+import {router} from './router';
+import bodyParser = require('koa-bodyparser');
+import mongoose = require('mongoose');
+import {logger} from './logger';
+import nconf = require('nconf');
+
+nconf.file('./config.json');
+
+mongoose.connect(nconf.get('dbURI'), {
```

Comment:
add `{ useUnifiedTopology: true }` to avaoid deprecated error

Reply:
done